css/sections/ai-uses.css

```
@@ -0,0 +1,130 @@
+	display: flex;
+	justify-content: space-between;
+	gap: var(--space-lg);
+	margin: 0 auto var(--space-xl) auto;
+}
+
+.aiUses-headings {
+	display: flex;
+	flex-direction: column;
+}
+
+.aiUses-headings h2 {
```

Stylelint (17.2.0)

[error] Expected class selector ".aiUses-headings" to be kebab-case (selector-class-pattern)

(selector-class-pattern)